final_task/rss_reader/Entry.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+
+class Entry:
+    """class for every article from http:link...link.rss"""
+
+    def __init__(self, title: str, date: str, article_link: str, summary: str, links: list):
+        self.title = title
+        self.date = date
+        self.article_link = article_link
+        self.summary = summary
+        self.links = links
+        self.parse_html()
```

Comment:
It is better to make this method to return parsed summary and write it like this:  
`self.summary = self.parse_html(summary)`  

Right now you assign `self.summary = summary` and after that modify it anyway. This assignment is a bit redundant

Reply:
fixed